### DISCUSSION TITLE
Some icons are missing

### DISCUSSION THREAD

**Discussion by K4LCIFER** [nextcloud/all-in-one#4376] [Questions] [answered]
### Steps to reproduce
1. Unknown. I just set up the container.

### Expected behavior 

I would expect the icons to all be present

### Actual behavior 

Some icons are missing — they only show up as little squares as shown:

![image](https://github.com/nextcloud/all-in-one/assets/7829231/d58680bb-161c-4524-ad54-e411995a1e2d)

### Host OS 

Ubuntu Server

#### Nextcloud AIO version 

v7.13.1

#### Current channel 

latest

#### Other valuable info 

Nextcloud Hub 7 v28.0.3

**Comment by szaimen** [comment, marked as answer]
Hi, I cannot reproduce this. Can you check if clearing the browser cache with `[ctrl]+[F5]` or `[shift]+[F5]` makes it work?
